components/camera/replaypcd/replaypcd.go

```diff
@@ -438,6 +463,9 @@ func (replay *pcdCamera) initCloudConnection(ctx context.Context) error {
 
 	replay.cloudConn = conn
 	replay.dataClient = dataServiceClient
+	replay.httpClient = &http.Client{
+		Timeout: downloadTimeout,
```

Comment:
is this necessary when the req has a context with this timeout as well?

Reply:
I do not believe it is necessary but I do not believe it hurts either

Reply:
It doesn't hurt, but be on the lookout for timeouts from both this and the context timeout if that behavior pops up. Now, two places can time out.

Reply:
+1